os/arch/arm/src/imxrt/imxrt_start.c

```diff
@@ -315,6 +368,12 @@ void __start(void)
 	const uint32_t *src;
 	uint32_t *dest;
 
+#ifdef CONFIG_ARMV7M_DTCM
+	imxrt_configure_dtcm();
```

Comment:
CONFIG_RAM_REGIONx_START has to be changed if CONFIG_ARMV7M_DTCM is enabled?

Reply:
CONFIG_RAM_REGIONx_START must always point to the start of the RAM region 0. So, yes.. it needs to be changed based on whether we use dtcm or ocram.